rollup.config.js

```diff
@@ -27,15 +27,16 @@ export default [
         only: ['tslib']
       }),
       typescriptPlugin({ typescript }),
+      invariantPlugin(),
       filesize(),
     ],
     onwarn,
   },
   {
     input: 'lib/react-apollo.esm.js',
     output: {
-      file: 'lib/react-apollo.umd.js',
-      format: 'umd',
```

Comment:
Is there a reason to fully remove the UMD? Sites like https://unpkg.com/ still look for these to index them so they can be required by CDN. Don't wanna come off as nosy but just trying to get some insights into things like these

Reply:
Ahh, I would be fine with keeping it if there's any value, and unpkg.com seems like a valid use case.